index.js

```diff
@@ -172,6 +172,7 @@ program
 
 program
   .command('build [root] [output]')
+  .option('<output>', 'put the generated files in the specified directory')
```

Comment:
As with earlier, commander.js expects a list of flags as the first argument to `.option()`. Passing in anything else doesn't seem to have meaningful results.

Reply:
Thanks for pointing out again. I will remove this line as well. For this issue, is there any other parts that i have missed which is supposed to be done?